modules/URL/ConferenceUrl.js

```diff
@@ -67,7 +67,14 @@ export default class ConferenceUrl {
      * Reloads the conference using original URL with all of the parameters.
      */
     reload() {
-        logger.info("Reloading the conference using URL: " + this.originalURL);
-        replace(this.originalURL);
+        logger.info(`Reloading the conference using URL: ${this.originalURL}`);
+
+        // Check if we are in an iframe and reload with the reload() utility
+        // because replace() is not working on an iframe.
+        if(window.self !== window.top) {
+            reload();
```

Comment:
In my opinion there's not much we can do. One option is to import everything but that will potentially bring a lot unused code. We can add some naming or comment but this means overhead during writing the code and also if there are some wrong comments or names this can be worse than return to check the imports.